src/scripts/ui/ExpandableMenu.js

```diff
@@ -47,6 +47,10 @@ class ExpandableMenu extends HtmlElement {
     this.mainInterior.append(htmlElement.element);
   }
   
+  addButton(button) {
+    this.element.prepend(button.getElement());
```

Comment:
I don't think the `prepend` lends to an intuitive codeflow, but for now we can keep it.